app/ports/management/commands/update-rsync.py

```diff
@@ -0,0 +1,46 @@
+import subprocess
+import json
+
+from django.core.management.base import BaseCommand, CommandError
+
+from parsing_scripts import git_update
+from ports.models import Port, Commit
+
+RSYNC = "rsync://rsync.macports.org/macports//trunk/dports/PortIndex_darwin_16_i386/PortIndex.json"
+JSON_FILE = "portindex.json"
```

Comment:
This one is to some extent "a local variable" (though leaving stuff around after being used), so I don't care that much if you leave the name here. What I do care though is to use the above mentioned global configuration file to store path to (semi)temporary files like this one.